lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -217,6 +223,18 @@ def self.is_firebasecmd_supported?(cmd)
         true
       end
 
+      def self.enable_access(app_id, release_id, emails, group_ids)
+        if emails.nil? && group_ids.nil?
```

Comment:
.present? Seems to give an error when I use it. I get an error message saying that the method was undefined. We will change the helper method to return nil if the string is empty to avoid any empty arrays.

Reply:
Ah, sorry turns out that a Rails only thing.